content/en/BestPractices/prevent-security-misconfig.md

```diff
@@ -0,0 +1,160 @@
+
+In the following example, we show how to prevent this problem for an Apache Tomcat web server:
+
+1. Open the `conf/web.xml` file in the directory where Tomcat is installed.
+
+     - In Windows 10, the default installation directory is `C:\Program Files (x86)\Apache Software Foundation\Tomcat 9.0`.
+
+1. In the file, set `<param-value>false</param-value>` to `false` for `<param-name>listings</param-name>`:
+
+     ```
+     <servlet>
+          <servlet-name>default</servlet-name>
+          <servlet-class>org.apache.catalina.servlets.DefaultServlet</servlet-class>
+          <init-param>
+               <param-name>debug</param-name>
+               <param-value>0</param-value>
+          </init-param>
+          <init-param>
+               <param-name>listings</param-name>
+               <param-value>false</param-value>
+          </init-param>
+          <load-on-startup>1</load-on-startup>
+     </servlet>
+     ```
+
+1. In the Apache web server, create a `.htaccess` file in the related directory.
+
+1. Add the following lines to the `httpd.conf` file, or replace existing lines with:
+
+     ```
+     <Directory /{YOUR DIRECTORY}>
+     Options FollowSymLinks
+     </Directory>
+     ```
+
+To turn directory listing off, modify the configuration files of your web server.
+
+### Default Accounts
+
+A developer may forget to remove default accounts when releasing the application.
+
+Examples of default accounts:
+
+- Application accounts
+- Web server logins
+- Cloud instances
+- Databases
+
+To prevent this:
+
+- Delete default accounts from the development environment once the development is complete
+- Change default passwords that come with installation packages
+- Restrict the exposure of administrator pages and web server logins in the production environment
+- Establish a strong password policy for your accounts
+
+### Files or Directories in a URL Path
+
+Sensitive files that may appear in a URL path include:
+
+- Backup files
+- Log files
+- Database files
+
+An attacker may discover sensitive files by brute-forcing the URL paths.
+
+To prevent this:
+
+- Remove any sensitive files or directories from the URL paths
+- Check your code before releasing it
+- To prevent URL fuzzing or a brute-force attack, apply a web application firewall and rate limiting checks
+
+### Misconfigured Cloud Services
+
+Misconfigurations in the cloud infrastructure or services may lead to data compromise. For example, an [Amazon S3 bucket](https://aws.amazon.com/s3/) with insufficient configuration may grant read and write access to an attacker.
+
+To prevent this:
+
+- Implement security automation to identify misconfigurations
+
+- Install updates and review your cloud security configurations on a regular basis
+
+## Best Practices
+
+- Follow a security checklist to prevent common misconfigurations
```

Comment:
Do you have an example of a well known "security checklist" that prevents misconfigurations?

Reply:
@mjang-cobalt  the idea here is to have a focussed checklist based on the application stack. developers can create a custom list where they can include all that is needed to be checked such as default logins, files in the URL path, debug, etc before pushing the application into production.

Reply:
So perhaps "Create and follow a security checklist" would make more sense? 

An example for the reader to start from could be really helpful.

Reply:
```suggestion
- Create and follow a security checklist
```